test/unit_tests/expression/test_aggregation.py

```diff
@@ -116,6 +116,58 @@ def test_aggregation_max(self):
         self.assertEqual(3, batch.frames.iloc[0][0])
         self.assertNotEqual(str(aggr_expr), None)
 
+    def test_aggregation_string_agg(self):
+        columnName = TupleValueExpression(name=0)
+        columnName.col_alias = 0
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_STRING_AGG, None, columnName
+        )
+        tuples = Batch(
+            pd.DataFrame(
+                {
+                    0: ["Hello", "World", "EvaDB", "Here"],
+                    1: ["Goodbye", "Everyone", "EvaDB", "Out"],
+                }
+            )
+        )
+        batch = aggr_expr.evaluate(tuples, delimiter=" ")
+        self.assertEqual("Hello World EvaDB Here", batch.frames.iloc[0][0])
+        self.assertNotEqual(str(aggr_expr), None)
+
+    def test_aggregation_string_agg_incorrect_column(self):
+        columnName = TupleValueExpression(name=0)
+        columnName.col_alias = 2
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_STRING_AGG, None, columnName
+        )
+        tuples = Batch(
+            pd.DataFrame(
+                {
+                    0: ["Hello", "World", "EvaDB" "Here"],
+                    1: ["Goodbye", "Everyone", "EvaDB" "Out"],
+                }
+            )
+        )
+        with pytest.raises(AssertionError):
+            aggr_expr.evaluate(tuples, delimiter=" ")
+
+    def test_aggregation_string_agg_incorrect_delimiter(self):
+        columnName = TupleValueExpression(name=0)
```

Comment:
Could you elaborate on the situation which this test is for? Thanks.

Reply:
Test if the delimiter argument is a string. Strings can only be concatenated with string delimiters